_episodes_rmd/03-day2.Rmd

```diff
@@ -1,82 +1,86 @@
 ---
 title: "Day 2: Introduction to Raster Data"
-teaching: 0
-exercises: 0
+exercises: 5
 questions:
-- "What is the coordinate reference system for the maximum temperature raster? Is it the same as the coordinate reference system for the minimum temperature raster? Visually explore the patterns in minimum and maximum monthly temperature differs across Wisconsin"
+- "What is the coordinate reference system for the maximum temperature raster? Is it the same as the coordinate reference system for the minimum temperature raster? Visually explore the patterns in minimum and maximum monthly temperature differs across Wisconsin.  Reproject the maxtemp raster into UTM coordinates"
 objectives:
-- "Importing raster data; checking coordinate reference system"
+- "Importing raster data; checking coordinate reference system; converting crs"
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "Always explore your raster first using the summary() and crs() functions. To plot your raster with ggplot(), first convert your raster to a data frame using the as.data.frame() function and specify the argument xy=TRUE. You may need to reproject your raster, for which you can use the projectRaster() function. There are many plotting and aesthetics options in the ggplot2 package for you to explore."
 ---
 
-
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
-solutions <- FALSE
+source("../bin/chunk-options.R")
+knitr_fig_path("03-")
 ```
 
-##Load in and explore the maximum and minimum temperature rasters for Wisconsin.
-
-```{r, echo=solutions, include=solutions}
-library(raster)
-library(ggplot2)
-library(rgdal)
-```
 
+## 1) Load in the maximum and minimum temperature rasters  for Wisconsin using the `raster()` function and explore using the `summary()` function.

```

Comment:
Maybe make this a shorter title and then use the full text below?  Example suggestion below.  Feel free to ignore or change as you prefer.

```suggestion
## 1 - Load and summarize

Practice the first steps of using data by loading in the maximum and minimum temperature rasters  for Wisconsin using the `raster()` function and then exploring using the `summary()` function.
```